pkg/plugins/user.go

```diff
@@ -205,11 +210,11 @@ func User(l logger.Interface, s schema.Stage, fs vfs.FS, console Console) error
 	for _, k := range users {
 		r := s.Users[k]
 		r.Name = k
-		if !s.Users[k].Exists() {
```

Comment:
`Exists()` was never `true` because the `Name` on the `s.Users[k]` wasn't set. It's set on `r` above though.

Reply:
nice catch